packages/shorebird_cli/integration_test/helpers.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:io';
+
+import 'package:mason_logger/mason_logger.dart';
+
+ProcessResult runCommand(
```

Comment:
Docs for this function (usage, what it does beyond invoking `Process.run`) would be helpful